XamarinCommunityToolkitSample/Pages/Views/SideMenuViewPage.xaml.cs

```diff
@@ -0,0 +1,9 @@
+﻿
+namespace Xamarin.CommunityToolkit.Sample.Pages.Views
+{
+	public partial class SideMenuViewPage
+	{
+		public SideMenuViewPage()
+			=> InitializeComponent();
```

Comment:
@jfversluis any ideas why tab is so long here, but not in, for example, ISideMenuList?

Reply:
Hm yeah interesting. Is it a GitHub viewing thing maybe?

Reply:
hm, maybe... but this is super strange.